src/components/home/Testimonials.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Flex, Grid, Heading, Text } from "@radix-ui/themes";
+
+import { partnerTestimonials } from "@/data/home";
+
+const Testimonials = () => {
+  return (
+    <Box p="7">
+      <Box className="text-center" mb="8">
+        <Heading as="h2" className="text-navy-900 text-6xl" m="4">
```

Comment:
Giving the Heading an idea goes along with the screen reader optimisation of the `<section>` further up
` id="testimonials-heading"`